public/src/pages/views/MemberView.tsx

```diff
@@ -82,11 +81,11 @@ export class MemberView extends React.Component<IUserViewerProps, IUserViewerSta
                 "Registered users",
                 this.state.acceptedUsers,
                 [],
-                ActionType.Menu,
+                ActionType.InRow,
                 (user: IUserRelation) => {
                     const links = [];
                     if (user.link.getStatus() === Enrollment.UserStatus.TEACHER) {
-                        links.push({ name: "This is a teacher", extra: "primary" });
+                        links.push({ name: "Teacher", extra: "light" });
                     } else {
                         links.push({ name: "Make Teacher", uri: "teacher", extra: "primary" });
                         links.push({ name: "Reject", uri: "reject", extra: "danger" });
```

Comment:
Suggest to change `Reject` with `Remove` (in the Registered Users list). For the Pending Users we should still keep `Reject`.

Reply:
Done.